Fleury/fleury.py

```diff
@@ -0,0 +1,32 @@
+#  This code is contributed by Riyazul555
+
+#  Time Complexity  =  O(V + E)  where V is the number of vertices and E is the number of edges.
+#  Space Complexity =  O(V + E)  where V is the number of vertices and E is the number of edges.
+
+def euler(nod, G, e, viz):
+    while G[nod]:
+        it = G[nod].pop()
+        if not viz[it[1]]:
+            viz[it[1]] = 1
+            euler(it[0], G, e, viz)
+    e.append(nod)
+
+if __name__ == "__main__":
+    input_file = sys.argv[1]
+    output_file = sys.argv[2]
+
+    with open(input_file, 'r') as fin, open(output_file, 'w') as fout:
+        n, m = map(int, fin.readline().split())
+        G = [[] for _ in range(100005)]
+
+        for _ in range(m):
```

Comment:
Generally, `_` serves as a throwaway variable in Python, but in your case, the variable is used, not ignored. Consider renaming it with a suggestive name.